atlp-rwanda/cabal-bn-be
# #181339544 find most traveled destinations

### What does this PR do?

Retrieves information about most traveled destinations

### Description of Task to be completed?
 
Users of Barefoot Nomad should be able to get info on the most frequently traveled center/destination

As users or a manager, they should be able to get information on the most traveled destination so that logical conclusions can be drawn.

### How should this be manually tested?

- Clone this [repo](https://github.com/atlp-rwanda/cabal-bn-be.git)

- Check out this branch `feature/181339544-most-traveled-destinations`

- Type **`yarn install`** to install dependencies, and **`yarn run dbmigrate:all`** to reinitialize the database.

- Use the postman to test the following endpoints

   PATCH

                 http://localhost:3000/api/v1/trips/{id}

                 Approve the trip status 

   GET

                 http://localhost:3000/api/v1/locations/most-visited


- Documentation: [Most Traveled Destinations](https://staging-bare-feature-18-voqgjg.herokuapp.com/api-docs/#/location/most%20visited%20locations)

         


### What are the relevant pivotal tracker stories?

[#181339544](https://www.pivotaltracker.com/n/projects/2555280/stories/181339544)


###Screenshots

<img width="1437" alt="Screen Shot 2022-04-04 at 14 02 20" src="https://user-images.githubusercontent.com/60324751/161539643-91249aa0-7647-4a1b-b121-e1dd9852a2ee.png">
